tests/pom.xml

```diff
@@ -579,6 +584,56 @@
         <hono.deviceregistry.mongodb.max-mem>150000000</hono.deviceregistry.mongodb.max-mem>
       </properties>
     </profile>
+    <profile>
+      <id>amqp-only</id>
```

Comment:
How about adding a `no-adapters` profile as well - for only running tests that are independent of the protocol adapters (e.g. the tests in `org.eclipse.hono.tests.registry`)?

Reply:
sure, why not ...